server/src/main/java/io/druid/client/CoordinatorServerView.java

```diff
@@ -158,10 +158,10 @@ private void serverRemovedSegment(DruidServerMetadata server, DataSegment segmen
     synchronized (lock) {
       log.debug("Removing segment[%s] from server[%s].", segmentId, server);
 
-      final SegmentLoadInfo segmentLoadInfo = segmentLoadInfos.get(segmentId);
+      SegmentLoadInfo segmentLoadInfo = segmentLoadInfos.get(segmentId);
       if (segmentLoadInfo == null) {
         log.warn("Told to remove non-existant segment[%s]", segmentId);
```

Comment:
Maybe should rephrase the logging statement now since and change level to "info"?